helpful.el

```diff
@@ -1675,7 +1664,7 @@ state of the current symbol."
        (format "This %s is advised." (downcase sym-type))))
 
     (let ((can-edebug
-           (helpful--can-edebug-p helpful--sym helpful--callable-p))
+           (helpful--can-edebug-p helpful--sym helpful--callable-p BUF POS OPENED))
```

Comment:
Would you mind using lower case here? Something like `helpful--callable-p buf pos opened`.